src/main/java/edu/harvard/iq/dataverse/engine/command/impl/RedetectFileTypeCommand.java

```diff
@@ -62,7 +63,7 @@ public DataFile execute(CommandContext ctxt) throws CommandException {
             }
 
             logger.fine("target file: " + localFile);
-            String newlyDetectedContentType = FileTypeDetection.determineFileType(localFile);
+            String newlyDetectedContentType = FileUtil.determineFileType(localFile, fileToRedetect.getDisplayName());
```

Comment:
Can we add a null check in the next line here? - From looking at #8821, it appears that determineFileType() may return null here, for especially unrecognizable files; and that results in a constraint violation.
Sorry for asking for this while the PR is already in QA; but would be a trivial change. Thanks.

Reply:
The change in FileUtil.java in this PR prevents a null return, so I think we're good?

Reply:
Oh, of course, doh! Thanks!